apps/website/src/app/api/ogimage/route.tsx

```diff
@@ -0,0 +1,275 @@
+/* eslint-disable import/no-default-export */
+/* eslint-disable @next/next/no-img-element */
+/* eslint-disable jsx-a11y/alt-text */
+
+import { ImageResponse } from '@vercel/og';
+
+// Utilities for title truncation
+// Counts the number of graphemes in a string
+const countGraphemes = (str: string): number => {
+  const segmenter = new Intl.Segmenter('ja', { granularity: 'grapheme' });
+  const segments = segmenter.segment(str);
+  return Array.from(segments).length;
+};
+
+// type OgProps = {
+//   title: string;
+//   cover?: string;
+//   authoricon?: string;
+//   authorname?: string;
+//   authorrole?: string;
+//   authorcount?: number;
+// };
+
+// Local custom fonts: https://vercel.com/docs/concepts/functions/edge-functions/og-image-generation/og-image-examples#using-a-custom-font
+// const domain = new URL(process.env['VERCEL_URL'] ? `https://${process.env['VERCEL_URL']}` : `http://localhost:${process.env['PORT'] || 3000}`);
+// const getFont = fetch(new URL('../../../core/font/NotoSansJP/NotoSansJP-Bold.woff', import.meta.url)).then((res) => res.arrayBuffer());
+
+// Local static images: https://vercel.com/docs/concepts/functions/edge-functions/og-image-generation/og-image-examples#using-a-local-image
+const getOgTemplateImage = fetch(new URL('../../../../public/og/og-template.png', import.meta.url)).then((res) => res.arrayBuffer());
+const getCoverImage = fetch(new URL('../../../../public/og/cover-astro.png', import.meta.url)).then((res) => res.arrayBuffer());
```

Comment:
`public` 使う場面結構多いので、いっそこうしちゃうのもあり
お手すきで

```suggestion
const getOgTemplateImage = fetch(new URL('#/public/og/og-template.png', import.meta.url)).then((res) => res.arrayBuffer());
const getCoverImage = fetch(new URL('#/public/og/cover-astro.png', import.meta.url)).then((res) => res.arrayBuffer());
```

Reply:
エイリアスを使うにしても、今の
```
"paths": {
      "@/*": [
        "./src/*"
      ]
    },
```
だと結局`@/../public/<path_to_the_asset>`みたいな感じになってあんまり意味ないし
変えるにしてもエイリアスの命名とか考えること多いし
`.tsconfig.json`に記述してもおそらく`fetch`のパスはランタイムで解決されるから`imports` https://nodejs.org/api/packages.html#packages_subpath_imports で設定しないといけなさそうで調べるのめんどそう
だから後回しで！